combat-trainer.lic

```diff
@@ -284,6 +276,21 @@ class LootProcess
     pause 0.2 until DRRoom.dead_npcs.empty?
   end
 
+  def consume(mob_noun, game_state, wounds)
+    check_thanatology(DRRoom.dead_npcs.first, 'consume', game_state)
+    # Flags.add('necro-spellcast', '^You feel fully prepared to cast your spell\.')
+    if game_state.casting
+      bput('release spell', '')
+      game_state.casting = false
+    end
+    bput("prepare cf", @settings.prep_messages)
+    game_state.casting = true
+    waitcastrt?
```

Comment:
How long does this wait typically take? Are you ok not taking any other actions in the meantime?

Since you've released the currently casting spell and inserted consume flesh (you should also insert a custom cast message), an alternative would be to exit the Process here and let SpellProcess take over the spellcasting. That would let other actions be taken while you wait for the spell to prep.